weblate/templates/accounts/contact.html

```diff
@@ -12,9 +12,9 @@
 
 <div class="alert alert-info">
   <ul>
-    <li>{% blocktrans %}For questions about specific translation project, contact its maintainers directly, not by this form.{% endblocktrans %}</li>
-    <li>{% blocktrans %}You can only contact maintainers of the whole server here.{% endblocktrans %}</li>
-    <li>{% blocktrans %}If you seek support for Weblate, or want to file a bug-report, check its website at {{ weblate_link }}.{% endblocktrans %}</li>
+    <li>{% blocktrans %}Contact maintainers of this Weblate here.{% endblocktrans %}</li>
```

Comment:
Ideally a placeholder for "contact maintainers of this _Fedora Weblate" here, but then again maybe some installations don't have "Weblate" in there.
"Placeholder" - "installation". would work